packages/ag-charts-enterprise/src/features/popover/popover.css

```diff
@@ -0,0 +1,32 @@
+.ag-charts-popover {
+    position: absolute;
+    display: grid;
+    grid: auto-flow auto / 1fr;
+    column-gap: 12px;
+    min-width: 200px;
+    background: var(--ag-charts-popover-background-color);
+    border: var(--ag-charts-popover-border);
+    border-radius: 4px;
+    padding: 6px 0;
+    overflow: hidden;
+}
+
+.ag-charts-popover:has(.ag-charts-popover__icon) {
+    grid: auto-flow auto / auto 1fr;
+}
+
+.ag-charts-popover__row {
+    display: grid;
+    grid-column: 1 / -1;
+    grid-template-columns: subgrid;
+    padding: 6px 12px;
+}
+
+.ag-charts-popover__row:focus {
```

Comment:
We tend to use :focus-visible, so that it applies when tabbing and not when clicking.

Reply:
This is actually a hover - we focus the hovered elements via JS